win32/src/winapi/user32/window.rs

```diff
@@ -622,6 +625,9 @@ pub enum SW {
 
 #[win32_derive::dllexport]
 pub async fn ShowWindow(machine: &mut Machine, hWnd: HWND, nCmdShow: Result<SW, u32>) -> bool {
+    // Store the show command for returning from GetWindowPlacement.
+    machine.state.user32.windows.get_mut(hWnd).unwrap().show_cmd = nCmdShow.clone();
```

Comment:
The reason nCmdShow is a Result<> is that we can't be sure the u32 coming in is a valid nCmdShow value, but that means you should just unwrap() it here and store the unwrapped SW enum in the window.